services/rpcstats/service.go

```diff
@@ -0,0 +1,44 @@
+package rpcstats
+
+import (
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// Service represents our own implementation of status status operations.
+type Service struct{}
+
+// New returns a new Service.
+func New() *Service {
+	return &Service{}
+}
+
+// APIs returns a list of new APIs.
+func (s *Service) APIs() []rpc.API {
+	return []rpc.API{
+		{
+			Namespace: "status",
```

Comment:
I think that an API called `status` in an app called **Status** should be something very central to the API. Would you consider `rpcstats` as an alternative name for the rpc status api namespace?

Reply:
Changed to `rpcstats` and so we have two methods `rpcstats_getStats` and `rpcstats_reset`.